CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,85 @@
+- Add user documentation for [WireGuard encryption]. ([#2902](https://github.com/antrea-io/antrea/pull/2902), [@jianjuns])
+- Add user documentation for [encap mode installation for EKS]. ([#2929](https://github.com/antrea-io/antrea/pull/2929), [@jianjuns])
+- Add the `nodePortLocal.enable` configuration parameter fot the Antrea Agent to enable NodePortLocal. ([#2924](https://github.com/antrea-io/antrea/pull/2924), [@antoninbas])
+
+### Changed
+
+- Remove chmod for OVSDB file from start_ovs, as the permissions are set correctly by OVS 2.15.1. ([#2803](https://github.com/antrea-io/antrea/pull/2803), [@antoninbas])
+- Reduce memory usage of antctl when collecting supportbundle. ([#2813](https://github.com/antrea-io/antrea/pull/2813), [@tnqn])
+- Do not perform SNAT for egress traffic to Kubernetes Node IPs. ([#2762](https://github.com/antrea-io/antrea/pull/2762), [@leonstack])
+- Change flow exporter export expiry mechanism to priority queue based, to reduce CPU usage and memory footprint. ([#2360](https://github.com/antrea-io/antrea/pull/2360), [@heanlan])
+- Send gratuitous ARP for EgressIP via the transport interface, as opposed to the interface with Node IP (if they are different). ([#2845](https://github.com/antrea-io/antrea/pull/2845), [@Jexf])
+- Wait for AntreaProxy to be ready before accessing Service if `antreaProxy.proxyAll` is enabled, to avoid connection issues on Agent startup. ([#2858](https://github.com/antrea-io/antrea/pull/2858), [@tnqn])
+- Ignore hostNetwork Pods selected by Egress, as they are not supported. ([#2851](https://github.com/antrea-io/antrea/pull/2851), [@Jexf])
+- Add windows task delay up to 30s to improve job resiliency (documentation) ([#2864](https://github.com/antrea-io/antrea/pull/2864), [@perithompson]) [Windows]
+- Remove offensive words from scripts and documentation. ([#2799](https://github.com/antrea-io/antrea/pull/2799), [@xiaoxiaobaba])
+- Make Pod labels optional in the flow records. By default, they will not be included in the flow records. Use the `recordContents.podLabels` configuration parameter for the Flow Aggregator to include them. ([#2739](https://github.com/antrea-io/antrea/pull/2739), [@yanjunz97])
+- Use readable name for OpenFlow tables. ([#2585](https://github.com/antrea-io/antrea/pull/2585), [@wenyingd])
+- Improve the OpenAPI schema for CRDs to validate the `matchExpressions` field. ([#2887](https://github.com/antrea-io/antrea/pull/2887), [@wenqiq])
+- Avoid duplicate processing of Egress. ([#2884](https://github.com/antrea-io/antrea/pull/2884), [@Jexf])
+- Fail fast if the source Pod for non-live-traffic Traceflow is invalid. ([#2736](https://github.com/antrea-io/antrea/pull/2736), [@gran-vmv])
+- Update [OVS pipeline documentation] to include information about AntreaProxy. ([#2725](https://github.com/antrea-io/antrea/pull/2725), [@hongliangl])
+- Ignore the IPs of kube-ipvs0 for Egress as they cannot be used for SNAT. ([#2930](https://github.com/antrea-io/antrea/pull/2930), [@Jexf])
```

Comment:
Is it a fix?

Reply:
The issue of the PR described a special scenario that egress IP is used for Service external IP as well and when kube-proxy uses ipvs mode.
The change can be considered a generic enhancement that can improve compatibility with kube-proxy and avoid unnecessary event processing. It's a little hard to describe the special scenario it can also fix with a few words and I think most users won't meet it, so I kept it as a change. 
Does it make sense to you?